identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```
@@ -527,3 +527,7 @@ QmFja3VwIENvZGUgRW5hYmxlZA__=Backup Code Enabled
 RW5hYmxlZCBBdXRoZW50aWNhdG9ycw__=Enabled Authenticators
 QmFja3VwIENvZGVz=Backup Codes
 RmFpbGVkIEJhY2t1cCBDb2RlIEF0dGVtcHRz=Failed Backup Code Attempts
+
+# SASS Product taglines
+sass.tagline=Create seamless login experiences for your applications in minutes
```

Let's fix the other places as well.

```suggestion
saas.tagline=Create seamless login experiences for your applications in minutes
```

Ref: https://www.salesforce.com/in/saas/

---

Addressed with 097ca7086a6c5bf616252953847fa63d22ef0b1e